ajax/libs/mathjax/package.json

```diff
@@ -21,6 +21,7 @@
       {
         "basePath": "",
         "files": [
+          "es5/**/!(node-main.js)",
```

Comment:
Looks like the other patterns will not be needed? Which file would be the main file then?

Reply:
We will be continuing to support version 2 for some time (version 3 is a major rewrite and has breaking changes), and expect to be issuing updates to version 2.  These will need to old file structure, which is why I didn't remove them.  That way both version 3 and version 2 will update properly.

Version 3 doesn't really have a main file, but I suppose `es5/startup.js` would be the best alternative for that.